solr/solr-ref-guide/src/common-query-parameters.adoc

```diff
@@ -250,18 +250,6 @@ This value is only checked at the time of:
 
 As this check is periodically performed, the actual time for which a request can be processed before it is aborted would be marginally greater than or equal to the value of `timeAllowed`. If the request consumes more time in other stages, custom components, etc., this parameter is not expected to abort the request. Regular search, JSON Facet and the Analytics component abandon requests in accordance with this parameter.
 
-== segmentTerminateEarly Parameter
```

Comment:
I don't think we should delete this documentation, I'd rather see something explaining what it has been replaced by and how that differs.